apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -32,76 +34,94 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
             displayName: team.displayName || '',
             interestGroupLeadershipRoleCount: getUniqueIdCount(
               team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                (item) =>
-                  item?.linkedFrom?.usersCollection?.items
-                    .filter(filterOutAlumni)
-                    .flatMap(flattenInterestGroupLeaders),
+                (teamMembershipItem) =>
+                  teamMembershipItem?.linkedFrom?.usersCollection?.items
+                    .flatMap(flattenInterestGroupLeaders)
+                    .filter(
+                      (item) =>
+                        item.interestGroupActive && item.userIsAlumni === false,
+                    )
+                    .map((item) => item.interestGroupId),
               ) || [],
             ),
             interestGroupMemberCount:
               (!team.inactiveSince &&
                 getUniqueIdCount([
                   ...(team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                    (item) =>
-                      item?.linkedFrom?.usersCollection?.items
-                        .filter(filterOutAlumni)
-                        .flatMap(flattenInterestGroupLeaders),
-                  ) || []),
-                  ...(team.linkedFrom?.interestGroupsCollection?.items.flatMap(
-                    (interestGroup) => interestGroup?.sys.id,
+                    (teamMembershipItem) =>
+                      teamMembershipItem?.linkedFrom?.usersCollection?.items
+                        .flatMap(flattenInterestGroupLeaders)
+                        .filter(
+                          (item) =>
+                            item.interestGroupActive &&
+                            item.userIsAlumni === false,
+                        )
+                        .map((item) => item.interestGroupId),
```

Comment:
This is the same as `interestGroupLeadershipRoleCount`, right? maybe it's better to extract it because if we need to change the way it is calculated we don't need to remember to change it in two separate places

Reply:
do you know what I think this is still such a small duplication that given the amount of requirements that we still have to implement I would hold off with such tiny refactoring, what do you think?

Reply:
ok :)